File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/base-file-attachment_item.ts


import { IBaseFileAttachment_Item } from './ibase-file-attachment_item';

export class BaseFileAttachment_Item implements IBaseFileAttachment_Item {
  
	public FileID: number = 0;
	public FileName: string = "";
	public FileSize: number = 0;
	public FileHash: number[] = null;
	public FileContent: number[] = null;
	public ModifiedByAppUserID: number = 0;
  
}

File: GTTS/INL.VettingService.Models/BaseFileAttachment_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class BaseFileAttachment_Item : IBaseFileAttachment_Item
    {
        public long FileID { get; set; }
        public string FileName { get; set; }
        public int FileSize { get; set; }
        public byte[] FileHash { get; set; }
        public byte[] FileContent { get; set; }
        public int ModifiedByAppUserID { get; set; }
    }
}
